android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -669,6 +673,55 @@ public void openURL(String url, ReadableMap options) {
         mActivity.startActivity(intent);
     }
 
+  @ReactMethod
+    public void getBranchQRCode(ReadableMap branchQRCodeSettingsMap, ReadableMap branchUniversalObjectMap, ReadableMap linkPropertiesMap, ReadableMap controlParamsMap, final Promise promise) {
+        
+        BranchUniversalObject branchUniversalObject = createBranchUniversalObject(branchUniversalObjectMap);
+        LinkProperties linkProperties = createLinkProperties(linkPropertiesMap, controlParamsMap);
+        BranchQRCode qrCode = createBranchQRCode(branchQRCodeSettingsMap);
+
+        try {
+            qrCode.getQRCodeAsData(getReactApplicationContext().getCurrentActivity(), branchUniversalObject, linkProperties, new BranchQRCode.BranchQRCodeDataHandler() {
+                @Override
+                public void onSuccess(byte[] qrCodeData) {
+                    String qrCodeString = Base64.encodeToString(qrCodeData, Base64.DEFAULT);
+                    promise.resolve(qrCodeString);
+                }
+    
+                @Override
+                public void onFailure(Exception e) {
+                    Log.d("Fail in main activity", String.valueOf(e));
```

Comment:
This log statement might not be accurate, maybe just
Log.d("getQRCodeAsData", e.getMessage())